package.json

```diff
@@ -1,21 +1,15 @@
 {
   "name": "react-native-webview",
   "description": "React Native WebView component for iOS, Android, macOS, and Windows",
-  "main": "index.js",
-  "typings": "index.d.ts",
   "author": "Jamon Holmgren <jamon@infinite.red>",
   "contributors": [
     "Thibault Malbranche <malbranche.thibault@gmail.com>"
   ],
   "license": "MIT",
```

Comment:
since we are changing the repo's folder structure, you will likely want to take a pass at updating the repo's CI to make sure the right commands run in the right folders